lib/main.dart

```diff
@@ -1,125 +1,22 @@
 import 'package:flutter/material.dart';
+import 'package:quiz_shower/ui/navigation/quiz_shower_navigation.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(const QuizShower());
 }
 
-class MyApp extends StatelessWidget {
-  const MyApp({super.key});
+class QuizShower extends StatelessWidget {
```

Comment:
SHOULD: 慣習的に `HogeApp` というクラス名にしたいので、 `QuizShowerApp` とかにしましょうかね
```suggestion
class QuizShowerApp extends StatelessWidget {
```

Reply:
りょーかいです！

Reply:
a6599a79e38aaec040da707856d144077c6f3ebf